src/electron/office/document_client.cc

```diff
@@ -397,6 +397,7 @@ v8::Local<v8::Value> DocumentClient::GotoOutline(int idx,
 base::span<char> DocumentClient::SaveToMemory(v8::Isolate* isolate) {
   char* pOutput = nullptr;
   size_t size = document_->saveToMemory(&pOutput, malloc);
+  SetView();
```

Comment:
This will probably fix the strange bug Teo was having in the editor->viewer toggle as well where it was saving the colorized PDF to memory (assuming it was a view issue) Nice!!